### PR TITLE
Add initial `eslint` config

### PR DESCRIPTION
**Pull request by orballo**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,16 +1,25 @@
 {
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
```
Comment (luisherranz) [inline .eslintrc]: I thought that if you use the `plugin:xxx/recommended` you don't need to add them in `plugins`.

Comment (orballo) [reply .eslintrc]: And you thought right. Thing is I tried like you say before, and for some reason it didn't work, but I have it working now.

Comment (luisherranz) [reply .eslintrc]: Super!

**Review by luisherranz — APPROVED** [review]
👍

**Pull request merged**